crates/holochain/tests/send_signal.rs

```diff
@@ -0,0 +1,136 @@
+use std::sync::Arc;
+
+use futures::StreamExt;
+use holochain::sweettest::{
+    SweetConductor, SweetConductorConfig, SweetDnaFile, SweetLocalRendezvous,
+};
+use holochain_types::signal::Signal;
+use holochain_wasm_test_utils::TestWasm;
+use holochain_websocket::WebsocketConfig;
+
+#[tokio::test(flavor = "multi_thread")]
+async fn send_signal() {
```

Comment:
Great test! I wonder if it deserves a doc comment or a more descriptive name so it's easier to see what it tests when people come across it later?

Reply:
It does. How about now?

Reply:
Great!